sources/north-america/us/oh/ODOT_TIMS_Road_Inventory.geojson

```diff
@@ -0,0 +1,1493 @@
+{
+  "type": "FeatureCollection",

```

Comment:
[The build is failing](https://github.com/osmlab/editor-layer-index/actions/runs/10069693122/job/27988861223?pr=2405):

> ERROR:root:Error in sources/north-america/us/oh/ODOT_TIMS_Road_Inventory.geojson : 'FeatureCollection' is not one of ['Feature']

You can fix this error by unnesting the one feature inside this feature collection.

Reply:
I've updated it thank you!

Reply:
You’ll need to delete everything except for the `Feature` object. There should be no `FeatureCollection` left.

Reply:
Thank you! Yeah I totally did that wrong the first time.